centreon/Makefile

```diff
@@ -0,0 +1,37 @@
+#
+# make all : will build node-external-reports.rpkg.
+# make licensed : will build a license limited version of the plugin
+#
+
+FILES = share/python/centreonapi bin/centreon-plugin.py etc/centreon.conf var/rudder/tmp/rudder_nodes.json etc/hooks.d/node-post-acceptance/centreon-post-acceptance.sh etc/hooks.d/node-pre-deletion/centreon-pre-deletion.sh
+SCRIPTS = postinst
+
+include ../makefiles/common-plugin.mk
+
+target/bin/centreon-plugin.py:
+	mkdir target/bin
+	cp centreon-plugin.py target/bin
+
+target/etc/centreon.conf:
+	mkdir target/etc
+	cp centreon.conf target/etc
+
+target/var/rudder/tmp/rudder_nodes.json:
+	mkdir target/var
+	mkdir target/var/rudder
+	mkdir target/var/rudder/tmp
+	cp rudder_nodes.json target/var/rudder/tmp
+
+target/etc/hooks.d/node-post-acceptance/centreon-post-acceptance.sh:
+	mkdir target/etc/hooks.d
+	mkdir target/etc/hooks.d/node-post-acceptance
+	cp centreon-post-acceptance.sh target/etc/hooks.d/node-post-acceptance
+
+target/etc/hooks.d/node-pre-deletion/centreon-pre-deletion.sh:
+	mkdir target/etc/hooks.d/node-pre-deletion
+	cp centreon-pre-deletion.sh target/etc/hooks.d/node-pre-deletion
+
+target/share/python/centreonapi:
+	mkdir target/share
+	mkdir target/share/python
+	cp -r centreonapi target/share/python/
```

Comment:
In fact you should just have the same directory structure in the repo as you want in the plugin and the make would just become a cp -r